### ISSUE TITLE
"Grimshot check" announces all required tools "OK", yet there is no "notify-send" or "jq"

### ISSUE THREAD

**Issue by scott092707** [swaywm/sway#5530]
scott@ASUS-Prime-B350M-A-CSM:~$ swaymsg -t get_version                                                                                                                                                                                                                                                                                                                                                                                 

sway version 1.4                                                                                                                                                                                                                                                                                                                                                                                                                       

- **Description:**
  - The steps you took in plain English to reproduce the problem.

Obtained grimshot (https://github.com/swaywm/sway/blob/master/contrib/grimshot)
Invoked it from the config file line:
bindsym Print exec .config/sway/grimshot.sh save area Desktop/$(date +'%Y%m%d_%H%M%S_Grimshot.png')

Works fine [Area, anyway], but ...

...when invoked from the terminal, it informs that notify-send was not found:
scott@ASUS-Prime-B350M-A-CSM:~$ .config/sway/grimshot.sh save area Desktop/$(date +'%Y%m%d_%H%M%S_Grimshot.png')
.config/sway/grimshot.sh: 33: notify-send: not found

...but the check option yields:

scott@ASUS-Prime-B350M-A-CSM:~$ .config/sway/grimshot.sh check
Checking if required tools are installed. If something is missing, install it to your system and make it available in PATH...
   grim: OK
   slurp: OK
   swaymsg: OK
   wl-copy: OK
   jq: OK
   notify-send: OK
   mktemp: OK

scott@ASUS-Prime-B350M-A-CSM:~$ notify-send                                                                                                                                                                                                                                                                                                                                                                                            
bash: notify-send: command not found                                                                                                                                                                                                                                                                                                                                                                                                   
scott@ASUS-Prime-B350M-A-CSM:~$ jq --help
bash: jq: command not found                                                                                                                                                                                                                                                                                                                                                                                                            


I don't know what package has notify-send, but the notification-related packages I have are:

scott@ASUS-Prime-B350M-A-CSM:~$ apt-cache policy libnotify4
libnotify4:                                                                                                                                                                                                                                                                                                                                                                                                                            
  Installed: 0.7.9-1                                                                                                                                                                                                                                                                                                                                                                                                                   
  Candidate: 0.7.9-1                                                                                                                                                                                                                                                                                                                                                                                                                   
  Version table:                                                                                                                                                                                                                                                                                                                                                                                                                       
 *** 0.7.9-1 500                                                                                                                                                                                                                                                                                                                                                                                                                       
        500 https://deb.debian.org/debian testing/main amd64 Packages                                                                                                                                                                                                                                                                                                                                                                  
        100 /var/lib/dpkg/status                                                                                                                                                                                                                                                                                                                                                                                                       
scott@ASUS-Prime-B350M-A-CSM:~$ apt-cache policy notification-daemon                                                                                                                                                                                                                                                                                                                                                                   
notification-daemon:                                                                                                                                                                                                                                                                                                                                                                                                                   
  Installed: 3.20.0-4                                                                                                                                                                                                                                                                                                                                                                                                                  
  Candidate: 3.20.0-4                                                                                                                                                                                                                                                                                                                                                                                                                  
  Version table:                                                                                                                                                                                                                                                                                                                                                                                                                       
 *** 3.20.0-4 500                                                                                                                                                                                                                                                                                                                                                                                                                      
        500 https://deb.debian.org/debian testing/main amd64 Packages                                                                                                                                                                                                                                                                                                                                                                  
        100 /var/lib/dpkg/status

**Comment by emersion** [comment]
This tool is in the [contrib directory](https://drewdevault.com/2020/06/06/Add-a-contrib-directory.html), which means they aren't officially supported. Feel free to send a patch though.

**Comment by scott092707** [comment]
`Feel free to send a patch though.`
Don't know how to do that...

I figured out that if I change line 55 from:
`if [ $? ]; then`
to 
`if [ $? -eq 0 ]; then`

then it works.  The original LOOKS like it ought to work, but it doesn't (much trial-and-error led to the eventual success).
With the change, all checks except for jq and notify-send are OK,  jq and notify-send are NOT FOUND, which is correct.

(also, there is a typo on line 9:
`##  - `jq`: json uliity to parse swaymsg output`
should probably be:
`##  - `jq`: json utility to parse swaymsg output`
)

[I tried to find out how to contact `Lauri / pokebadgerswithspoon` (the apparent single contributor) to have him/her change
his/her own work, but was unable to figure out how to do that.]
